src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -0,0 +1,126 @@
+package core.basesyntax;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.OffsetDateTime;
+import java.util.Optional;
+
+public class JavaDateTimeApi {
+
+    /**
+     * Верните текущую дату в виде строки в зависимости от запроса.
+     *
+     * @param datePart Запрос на часть даты или всю дата целиком:
+     *                 - FULL - текущая дата целиком год, месяц, день (число месяца),
+     *                 возвращаемое значение по умолчанию;
+     *                 - YEAR - текущий год;
+     *                 - MONTH - название текущего месяца;
+     *                 - DAY - текущий день (число месяца);
+     **/
+    public String todayDate(DateTimePart datePart) {
+        return "Today";
+    }
+
+    /**
+     * Верните Optional даты соответствующей дате в массиве.
+     *
+     * @param dateParams Дан массив данных состоящий из 3-х элементов, где:
+     *                   - 1-й элемент массива - год;
+     *                   - 2-й элемент массива - месяц;
+     *                   - 3-й элемент массива - день (число);
+     **/
+
+    public Optional<LocalDate> someDate(Integer[] dateParams) {
```

Comment:
@nvkaip Назва методу не сильно інформативна
```suggestion
    public Optional<LocalDate> getDate(Integer[] dateParams) {
```